css/src/components/persona.scss

```diff
@@ -0,0 +1,203 @@
+$persona-avatar-xs: 28px !default;
+$persona-avatar-sm: 32px !default;
+$persona-avatar-md: 36px !default;
+$persona-avatar-lg: 42px !default;
+$persona-avatar-xl: 48px !default;
+
+$persona-avatar-font-size: 0.45em;
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-avatar-border-radius: $border-radius-rounded;
+
+$persona-name-font-size: $font-size-7 !default;
+$persona-name-font-color: $text !default;
+
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+
+$persona-gap-size: $layout-1 !default;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+	font-size: $persona-avatar-md;
+
+	.persona-avatar {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: $persona-avatar-md;
+		height: $persona-avatar-md;
+		border-radius: $persona-avatar-border-radius;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		font-size: $persona-avatar-font-size;
+		line-height: 1;
+		text-transform: uppercase;
+
+		img,
+		svg {
+			display: block;
+			width: 100%;
+			height: auto;
+			max-height: 100%;
+			border-radius: $persona-avatar-border-radius;
+		}
+	}
+
+	.persona-details {
+		align-self: center;
+		color: $persona-metadata-font-color;
+		font-size: $persona-metadata-font-size;
+
+		.persona-name {
+			font-size: $persona-name-font-size;
+
+			&:not(a) {
+				color: $persona-name-font-color;
+			}
+		}
+	}
+
+	// Sizes
+
+	&.persona-xs {
+		font-size: $persona-avatar-xs;
+
+		.persona-avatar {
+			width: $persona-avatar-xs;
+			height: $persona-avatar-xs;
+		}
+	}
+
+	&.persona-sm {
+		font-size: $persona-avatar-sm;
+
+		.persona-avatar {
+			width: $persona-avatar-sm;
+			height: $persona-avatar-sm;
+		}
+	}
+
+	&.persona-md {
+		font-size: $persona-avatar-md;
+
+		.persona-avatar {
+			width: $persona-avatar-md;
+			height: $persona-avatar-md;
+		}
+	}
+
+	&.persona-lg {
+		font-size: $persona-avatar-lg;
+
+		.persona-avatar {
+			width: $persona-avatar-lg;
+			height: $persona-avatar-lg;
+		}
+	}
+
+	&.persona-xl {
+		font-size: $persona-avatar-xl;
+
+		.persona-avatar {
+			width: $persona-avatar-xl;
+			height: $persona-avatar-xl;
+		}
+	}
+
+	@include tablet {
```

Comment:
responsive sizes need initially? seems like we're adding a lot of cruft here?

Reply:
Removed, I think I picked it up from the image and after looking around - we don't really need that much flexibility.